test/support/feature_helpers.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Community.FeatureHelpers do
+  use Hound.Helpers
+
+  def click_on(text) do
+    find_element(:link_text, text)
+    |> click
+  end
+
+  def click_role(name) do
+    find_element(:css, "[data-role=#{name}]")
+    |> click
+  end
+
+  def fill_in(type, field_name, text) do
```

Comment:
Right now these calls are a bit hard to understand. What do you thin of doing something like

```elixir
def fill_in(type, field_name, with: text) do
```

I think that might read a little better